### ISSUE TITLE
Windows test failures

### ISSUE THREAD

**Issue by gpa14** [hlamer/enki#383]
Windows 8.1, run_all.py, WinPython 3.4.3.7Qt5
Revision ac6650df386d2804667792607db4887b59de95d1
-unable to capture more output
```
    self.assertTrue(self.sawSignal)
AssertionError: False is not true

======================================================================
FAIL: test_sync20b (test_plugins.test_preview_sync.Test)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\Users\Grant\Documents\CodeChat\software\enki\tests\base.py", line 115
, in wrapper
    raise value.with_traceback(tracebackObj)
  File "C:\Users\Grant\Documents\CodeChat\software\enki\tests\base.py", line 88,
 in execWithArgs
    func(*args)
  File "C:\Users\Grant\Documents\CodeChat\software\enki\tests\test_lib\..\test_p
lugins\test_preview_sync.py", line 414, in test_sync20b
    self._doBasicTest('rst')
  File "C:\Users\Grant\Documents\CodeChat\software\enki\tests\test_plugins\test_
preview.py", line 152, in _doBasicTest
    self.testText), numEmittedExpected=numEmittedExpected)
  File "C:\Users\Grant\Documents\CodeChat\software\enki\tests\test_plugins\test_
preview.py", line 146, in _assertHtmlReady
    start()
  File "C:\Users\Grant\Documents\CodeChat\software\enki\tests\base.py", line 584
, in __exit__
    self.assertTrue(self.sawSignal)
AssertionError: False is not true

======================================================================
FAIL: test_sync20c (test_plugins.test_preview_sync.Test)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\Users\Grant\Documents\CodeChat\software\enki\tests\base.py", line 115
, in wrapper
    raise value.with_traceback(tracebackObj)
  File "C:\Users\Grant\Documents\CodeChat\software\enki\tests\base.py", line 88,
 in execWithArgs
    func(*args)
  File "C:\Users\Grant\Documents\CodeChat\software\enki\tests\test_lib\..\test_p
lugins\test_preview_sync.py", line 431, in test_sync20c
    self._doBasicTest('rst')
  File "C:\Users\Grant\Documents\CodeChat\software\enki\tests\test_plugins\test_
preview.py", line 152, in _doBasicTest
    self.testText), numEmittedExpected=numEmittedExpected)
  File "C:\Users\Grant\Documents\CodeChat\software\enki\tests\test_plugins\test_
preview.py", line 146, in _assertHtmlReady
    start()
  File "C:\Users\Grant\Documents\CodeChat\software\enki\tests\base.py", line 584
, in __exit__
    self.assertTrue(self.sawSignal)
AssertionError: False is not true

======================================================================
FAIL: test_sync21 (test_plugins.test_preview_sync.Test)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\Users\Grant\Documents\CodeChat\software\enki\tests\base.py", line 115
, in wrapper
    raise value.with_traceback(tracebackObj)
  File "C:\Users\Grant\Documents\CodeChat\software\enki\tests\base.py", line 88,
 in execWithArgs
    func(*args)
  File "C:\Users\Grant\Documents\CodeChat\software\enki\tests\test_lib\..\test_p
lugins\test_preview_sync.py", line 447, in test_sync21
    self._doBasicTest('rst')
  File "C:\Users\Grant\Documents\CodeChat\software\enki\tests\test_plugins\test_
preview.py", line 152, in _doBasicTest
    self.testText), numEmittedExpected=numEmittedExpected)
  File "C:\Users\Grant\Documents\CodeChat\software\enki\tests\test_plugins\test_
preview.py", line 146, in _assertHtmlReady
    start()
  File "C:\Users\Grant\Documents\CodeChat\software\enki\tests\base.py", line 584
, in __exit__
    self.assertTrue(self.sawSignal)
AssertionError: False is not true

======================================================================
FAIL: test_sync2a (test_plugins.test_preview_sync.Test)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\Users\Grant\Documents\CodeChat\software\enki\tests\base.py", line 115
, in wrapper
    raise value.with_traceback(tracebackObj)
  File "C:\Users\Grant\Documents\CodeChat\software\enki\tests\base.py", line 88,
 in execWithArgs
    func(*args)
  File "C:\Users\Grant\Documents\CodeChat\software\enki\tests\test_lib\..\test_p
lugins\test_preview_sync.py", line 108, in test_sync2a
    self._testSyncString('T')
  File "C:\Users\Grant\Documents\CodeChat\software\enki\tests\test_lib\..\test_p
lugins\test_preview_sync.py", line 90, in _testSyncString
    self._doBasicTest('rst')
  File "C:\Users\Grant\Documents\CodeChat\software\enki\tests\test_plugins\test_
preview.py", line 152, in _doBasicTest
    self.testText), numEmittedExpected=numEmittedExpected)
  File "C:\Users\Grant\Documents\CodeChat\software\enki\tests\test_plugins\test_
preview.py", line 146, in _assertHtmlReady
    start()
  File "C:\Users\Grant\Documents\CodeChat\software\enki\tests\base.py", line 584
, in __exit__
    self.assertTrue(self.sawSignal)
AssertionError: False is not true

======================================================================
FAIL: test_sync3 (test_plugins.test_preview_sync.Test)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\Users\Grant\Documents\CodeChat\software\enki\tests\base.py", line 115
, in wrapper
    raise value.with_traceback(tracebackObj)
  File "C:\Users\Grant\Documents\CodeChat\software\enki\tests\base.py", line 88,
 in execWithArgs
    func(*args)
  File "C:\Users\Grant\Documents\CodeChat\software\enki\tests\test_lib\..\test_p
lugins\test_preview_sync.py", line 120, in test_sync3
    self._testSyncString(self.testText)
  File "C:\Users\Grant\Documents\CodeChat\software\enki\tests\test_lib\..\test_p
lugins\test_preview_sync.py", line 90, in _testSyncString
    self._doBasicTest('rst')
  File "C:\Users\Grant\Documents\CodeChat\software\enki\tests\test_plugins\test_
preview.py", line 152, in _doBasicTest
    self.testText), numEmittedExpected=numEmittedExpected)
  File "C:\Users\Grant\Documents\CodeChat\software\enki\tests\test_plugins\test_
preview.py", line 146, in _assertHtmlReady
    start()
  File "C:\Users\Grant\Documents\CodeChat\software\enki\tests\base.py", line 584
, in __exit__
    self.assertTrue(self.sawSignal)
AssertionError: False is not true

======================================================================
FAIL: test_sync4 (test_plugins.test_preview_sync.Test)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\Users\Grant\Documents\CodeChat\software\enki\tests\base.py", line 115
, in wrapper
    raise value.with_traceback(tracebackObj)
  File "C:\Users\Grant\Documents\CodeChat\software\enki\tests\base.py", line 88,
 in execWithArgs
    func(*args)
  File "C:\Users\Grant\Documents\CodeChat\software\enki\tests\test_lib\..\test_p
lugins\test_preview_sync.py", line 146, in test_sync4
    self._sendJsClick(0)
  File "C:\Users\Grant\Documents\CodeChat\software\enki\tests\test_lib\..\test_p
lugins\test_preview_sync.py", line 130, in _sendJsClick
    self._doBasicTest('rst')
  File "C:\Users\Grant\Documents\CodeChat\software\enki\tests\test_plugins\test_
preview.py", line 152, in _doBasicTest
    self.testText), numEmittedExpected=numEmittedExpected)
  File "C:\Users\Grant\Documents\CodeChat\software\enki\tests\test_plugins\test_
preview.py", line 146, in _assertHtmlReady
    start()
  File "C:\Users\Grant\Documents\CodeChat\software\enki\tests\base.py", line 584
, in __exit__
    self.assertTrue(self.sawSignal)
AssertionError: False is not true

======================================================================
FAIL: test_sync5 (test_plugins.test_preview_sync.Test)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\Users\Grant\Documents\CodeChat\software\enki\tests\base.py", line 115
, in wrapper
    raise value.with_traceback(tracebackObj)
  File "C:\Users\Grant\Documents\CodeChat\software\enki\tests\base.py", line 88,
 in execWithArgs
    func(*args)
  File "C:\Users\Grant\Documents\CodeChat\software\enki\tests\test_lib\..\test_p
lugins\test_preview_sync.py", line 152, in test_sync5
    self._sendJsClick(8)
  File "C:\Users\Grant\Documents\CodeChat\software\enki\tests\test_lib\..\test_p
lugins\test_preview_sync.py", line 130, in _sendJsClick
    self._doBasicTest('rst')
  File "C:\Users\Grant\Documents\CodeChat\software\enki\tests\test_plugins\test_
preview.py", line 152, in _doBasicTest
    self.testText), numEmittedExpected=numEmittedExpected)
  File "C:\Users\Grant\Documents\CodeChat\software\enki\tests\test_plugins\test_
preview.py", line 146, in _assertHtmlReady
    start()
  File "C:\Users\Grant\Documents\CodeChat\software\enki\tests\base.py", line 584
, in __exit__
    self.assertTrue(self.sawSignal)
AssertionError: False is not true

======================================================================
FAIL: test_sync6 (test_plugins.test_preview_sync.Test)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\Users\Grant\Documents\CodeChat\software\enki\tests\base.py", line 115
, in wrapper
    raise value.with_traceback(tracebackObj)
  File "C:\Users\Grant\Documents\CodeChat\software\enki\tests\base.py", line 88,
 in execWithArgs
    func(*args)
  File "C:\Users\Grant\Documents\CodeChat\software\enki\tests\test_lib\..\test_p
lugins\test_preview_sync.py", line 158, in test_sync6
    self._sendJsClick(len(self.testText))
  File "C:\Users\Grant\Documents\CodeChat\software\enki\tests\test_lib\..\test_p
lugins\test_preview_sync.py", line 130, in _sendJsClick
    self._doBasicTest('rst')
  File "C:\Users\Grant\Documents\CodeChat\software\enki\tests\test_plugins\test_
preview.py", line 152, in _doBasicTest
    self.testText), numEmittedExpected=numEmittedExpected)
  File "C:\Users\Grant\Documents\CodeChat\software\enki\tests\test_plugins\test_
preview.py", line 146, in _assertHtmlReady
    start()
  File "C:\Users\Grant\Documents\CodeChat\software\enki\tests\base.py", line 584
, in __exit__
    self.assertTrue(self.sawSignal)
AssertionError: False is not true

======================================================================
FAIL: test_sync7 (test_plugins.test_preview_sync.Test)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\Users\Grant\Documents\CodeChat\software\enki\tests\base.py", line 115
, in wrapper
    raise value.with_traceback(tracebackObj)
  File "C:\Users\Grant\Documents\CodeChat\software\enki\tests\base.py", line 88,
 in execWithArgs
    func(*args)
  File "C:\Users\Grant\Documents\CodeChat\software\enki\tests\test_lib\..\test_p
lugins\test_preview_sync.py", line 167, in test_sync7
    self.check_sync1()
  File "C:\Users\Grant\Documents\CodeChat\software\enki\tests\test_lib\..\test_p
lugins\test_preview_sync.py", line 54, in check_sync1
    self._doBasicTest('rst')
  File "C:\Users\Grant\Documents\CodeChat\software\enki\tests\test_plugins\test_
preview.py", line 152, in _doBasicTest
    self.testText), numEmittedExpected=numEmittedExpected)
  File "C:\Users\Grant\Documents\CodeChat\software\enki\tests\test_plugins\test_
preview.py", line 146, in _assertHtmlReady
    start()
  File "C:\Users\Grant\Documents\CodeChat\software\enki\tests\base.py", line 584
, in __exit__
    self.assertTrue(self.sawSignal)
AssertionError: False is not true

======================================================================
FAIL: test_sync8 (test_plugins.test_preview_sync.Test)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\Users\Grant\Documents\CodeChat\software\enki\tests\base.py", line 115
, in wrapper
    raise value.with_traceback(tracebackObj)
  File "C:\Users\Grant\Documents\CodeChat\software\enki\tests\base.py", line 88,
 in execWithArgs
    func(*args)
  File "C:\Users\Grant\Documents\CodeChat\software\enki\tests\test_lib\..\test_p
lugins\test_preview_sync.py", line 183, in test_sync8
    self._doBasicTest('rst')
  File "C:\Users\Grant\Documents\CodeChat\software\enki\tests\test_plugins\test_
preview.py", line 152, in _doBasicTest
    self.testText), numEmittedExpected=numEmittedExpected)
  File "C:\Users\Grant\Documents\CodeChat\software\enki\tests\test_plugins\test_
preview.py", line 146, in _assertHtmlReady
    start()
  File "C:\Users\Grant\Documents\CodeChat\software\enki\tests\base.py", line 584
, in __exit__
    self.assertTrue(self.sawSignal)
AssertionError: False is not true

======================================================================
FAIL: test_sync9 (test_plugins.test_preview_sync.Test)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\Users\Grant\Documents\CodeChat\software\enki\tests\base.py", line 115
, in wrapper
    raise value.with_traceback(tracebackObj)
  File "C:\Users\Grant\Documents\CodeChat\software\enki\tests\base.py", line 88,
 in execWithArgs
    func(*args)
  File "C:\Users\Grant\Documents\CodeChat\software\enki\tests\test_lib\..\test_p
lugins\test_preview_sync.py", line 234, in test_sync9
    self._textToWeb('ne')
  File "C:\Users\Grant\Documents\CodeChat\software\enki\tests\test_lib\..\test_p
lugins\test_preview_sync.py", line 205, in _textToWeb
    self._doBasicTest('rst')
  File "C:\Users\Grant\Documents\CodeChat\software\enki\tests\test_plugins\test_
preview.py", line 152, in _doBasicTest
    self.testText), numEmittedExpected=numEmittedExpected)
  File "C:\Users\Grant\Documents\CodeChat\software\enki\tests\test_plugins\test_
preview.py", line 146, in _assertHtmlReady
    start()
  File "C:\Users\Grant\Documents\CodeChat\software\enki\tests\base.py", line 584
, in __exit__
    self.assertTrue(self.sawSignal)
AssertionError: False is not true

----------------------------------------------------------------------
Ran 298 tests in 596.140s

FAILED (failures=38, skipped=24, expected failures=3)
```

**Comment by bjones1** [comment]
I can reproduce this. I'll take a look...

**Comment by hlamer** [comment]
@bjones1, will you also check qutepart #40?

**Comment by bjones1** [comment]
Will do.

**Comment by bjones1** [comment]
Fixed by 3b72e4fc08d1d9bba5cf538ff239be22eff2b376.
